docs/scanner-quickstart.md

```diff
@@ -191,18 +191,23 @@ trace:
 
 ## Register Scan Node
 
-To register your node to the registry contract, you can run `forta register --owner-address <address>`.
+Your scan node has an Ethereum address that makes two main features possible:
+  - Receiving detection bots to run
+  - Asserting an authority on the outputted alerts
+  
+While this address remains as the main identity, it is ownable and manageable by a different wallet. The owner wallet allows you to disable your scan node later and avoid slashing while you do maintenance (for a short period) or when you decide to shut down your node entirely. The owner wallet address needs to be a different address than your scan node address.
```

Comment:
Maybe it's easier to explain that the Scanner is:
- Identified by the scanner address
- After registration, a NFT ERC721 is emitted to the owner wallet. The owner wallet can disable and enable the scanner